Frontend/ui-driver/src/Services/Endpoint.purs

```diff
@@ -378,3 +378,7 @@ registerAadhaar _ = (getBaseUrl "") <> "/driver/register/aadhaarCard"
 
 verifyUPI :: String -> String
 verifyUPI dummy = (getBaseUrl "") <> "/driver/profile/verify/vpaStatus"
+
+
+callOnClick :: String -> String
+callOnClick rideId =  (getBaseUrl "") <> "/exotel/call/ride/" <> rideId <> "/call/onClick"
```

Comment:
can we rename it to `exotelCallEvent`/`exotelRideCallEvent` as per behaviour, for user as well